.github/workflows/try-runtime.yml

```diff
@@ -0,0 +1,40 @@
+name: Run try-runtime on storage migration label
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  run_if_label_matches:
+    if: github.event.label.name == 'migration'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+
+      - name: Setup worker
+        uses: "./.github/templates/setup-worker"
+
+      - name: Cache Build artefacts
+        uses: actions/cache@v3
```

Comment:
While you can benefit from using cache here, you are doing a very specific build here. So I suggest that you do a cache [restore](https://github.com/actions/cache/blob/main/restore/README.md) here without saving the resulting artifacts